canopen_chain_node/src/ros_chain.cpp

```diff
@@ -593,7 +593,7 @@ RosChain::~RosChain(){
         LayerStatus s;
         halt(s);
         shutdown(s);
-    }catch(...){ LOG("CATCH"); }
+    }catch(...){ ROS_ERROR("CATCH"); }
```

Comment:
What is the logic here? canopen_chain_node, canopen_motor_node and socketcan_bridge use `ROS_*`-macros directly while the ros-agnostic packages use rosconsole's `ROSCANOPEN_*` macros? Please add a sentence to CONTRIBUTING.md or similar explaining the idea to other developers!

Reply:
>  canopen_chain_node, canopen_motor_node and socketcan_bridge use ROS_*-macros directly while the ros-agnostic packages use rosconsole's ROSCANOPEN_* macros

Exactly.

> Please add a sentence to CONTRIBUTING.md or similar explaining the idea to other developers!

Good idea.

Reply:
This is now tracked in #345